docs/guide/setup-examples/julia.md

```diff
@@ -0,0 +1,401 @@
+We will now have multiple commands for different versions of Julia.
+So in the Command Palette either choose **Terminus: Open Julia Stable**
+or **Terminus: Open Julia Nightly**.
+Note the magic incantation to allow for a space in the Julia-development path:
+the quoting `"%LOCALAPPDATA%\\Programs\\\"Julia 1.5.0-DEV\"\\bin\\julia.exe"`
+is needed to make it work.
+
+It is possible to set environment variables like `JULIA_NUM_THREADS` here.
+
+Finally, the editor gives focus to the top-most file in the group 1.
+This is useful when two columns or two rows are used for the layout.
+Opening Julia with the cursor in a source file
+then places the REPL in the other view.
+
+### Snippets
+
+The [Julia][] package comes with predefined snippets.
+I have defined some of my own, such as this one
+to speed up the specification of a doc string (file `docstring.sublime-snippet`):
+```
+<snippet>
+    <content><![CDATA[
+"""
+    ${1:bar(x[, y])}
+
+${0:Compute}
+"""
+]]></content>
+    <tabTrigger>docs</tabTrigger>
+    <scope>source.julia</scope>
+</snippet>
+```
+The snippets go into one file per snippet, which I put in `USER\snippets`.
+You can find more snippets on [Petr Github][].k
+
+[Petr Github]: https://github.com/PetrKryslUCSD/HowToUseJuliaWithSublimeText3/tree/master/snippets
+
+## Usage
+
+### Open terminal
+
+Bring up the **Command Palette**, start typing in `Terminus`.
+Select **Terminus: List Shells**,
+and from the list that appears choose the shell you wish to start.
+Note that then you can select whether to start the shell
+in a panel at the bottom or in a separate view.
+
+Julia may be started in the resulting terminal in the usual way.
+The terminals that I have checked out (`cmd` and the WSL Ubuntu shell)
+work as expected.
+
+### Open a source file and then run Julia from the source file
+
+Open a Julia source file,
+and in the **Command Palette**, start typing in `Terminus`.
+Select **Terminus: Open Julia**.
+This will open the default terminal on the user's platform
+(`cmd` on Windows, but see also the note about the Git bash below),
+and run Julia in the directory that contains the open file.
+
+### Evaluating code
+
+Select some Julia code and press <Key k="ctrl+enter" />.
+The code will be pasted into the **Terminus** window
+and evaluated in Julia (assuming Julia was started in that **Terminus** window).
+This also works for evaluating a line of code:
+place the cursor on a line and type <Key k="ctrl+enter" />.
+
+#### Running Julia files
+
+In a opened Julia source file, press <Key k="ctrl+b" />
+(which is the key binding for **Tools → Build**).
+The current file will be evaluated in a Julia-running Terminus window
+with an `include()`.
+
+### Asking for help in the REPL
+
+From a Julia source file, press <Key k="ctrl+shift+x, ctrl+h"> to ask the Julia
+REPL to display help about the current selection.
+
+## Zeal
+
+Zeal is an open-source browser of off-line documentation.
```

Comment:
You should mention that Zeal only supports Linux and Windows (since macOS is handled by Dash).